src/SFA.DAS.Reservations.Web/Controllers/ReservationsController.cs

```diff
@@ -146,14 +146,19 @@ public async Task<IActionResult> Review(ReservationsRouteModel routeModel)
                 RouteNames.EmployerCreateReservation : 
                 RouteNames.ProviderCreateReservation;
 
-            var changeRouteName = routeModel.UkPrn == null ? 
+            var changeCourseRouteName = routeModel.UkPrn == null ? 
```

Comment:
WIll probably need to look at merging this into MF-14

Reply:
ya, let it go first and I'll pick up the pieces.